### PR TITLE
Merge branch 'development' of https://github.com/DamienButterworth/sc…

### PR DESCRIPTION
**Pull request by DamienButterworth**
…ala into Day2

# Conflicts:
#	src/main/scala/Week 1 Day 1.sc

Completed Upto Garage and started implementation on Garage
Continued Implementation on Garage

### CODE REVIEW

File: src/main/scala/Garage/Garage.scala [hunk 1]
```diff
@@ -2,10 +2,11 @@ package Garage
 
 import scala.collection.mutable.ArrayBuffer
 
-class Garage {
-
-  var customers = ArrayBuffer[Customer]()
+case class Garage() {
+  var openGarage = false
+  var customers:ArrayBuffer[Customer] = ArrayBuffer[Customer]()
   def addCustomer(customer:Customer):Unit = if (customers.count(value => value.vehicle.registrationNumber == customer.vehicle.registrationNumber) == 0) customers += customer
```
Comment (mariannepearson0) [inline src/main/scala/Garage/Garage.scala]: Bit long, line should fit on screen

File: src/main/scala/Garage/Garage.scala [hunk 2]
```diff
@@ -2,10 +2,11 @@ package Garage
...
   def removeCustomer(customer: Customer):Unit = if (customers.contains(customer)) customers.remove(customers.indexOf(customer))
   def getCustomers(regNumber:String):Option[Customer] = customers.find(value => value.vehicle.registrationNumber == regNumber)
```
Comment (mariannepearson0) [inline src/main/scala/Garage/Garage.scala]: Could use wildcards for find function

File: src/main/scala/Garage/GarageSystem.scala [hunk 3]
```diff
@@ -3,33 +3,30 @@ package Garage
...
+        println(garage.getCustomers("ST03GPV").get.getTotal)
+      } else {
+        println("Customer Not Found With That Registration")
+      }
+    }
+  else
     {
-      println(garage.getCustomers("ST03GPV").get.getTotal)
-    } else {
-    println("Customer Not Found With That Registration")
```
Comment (mariannepearson0) [inline src/main/scala/Garage/GarageSystem.scala]: Lots of if/elses use pattern matching?

**Comment by tvaidotas** [comment]
:+1:

**Pull request merged**